aspnetcore/mvc/views/razor.md

```diff
@@ -516,6 +536,29 @@ The following code is the generated Razor C# class:
 
 [!code-csharp[](razor/sample/Classes/Views_Home_Test_cshtml.cs?range=1-19)]
 
+You can also use markup in `@functions` to server as templating methods:
```

Comment:
```suggestion
`@functions` methods serve as templating methods when they return markup:
```

Reply:
They don't technically return markup so i'll say when they have markup

Reply:
Side note: "Have" is ok here. Alternatives to a state verb (e.g., be, has, have) convey a more *action*: contain, include, possess, enclose.